helm/charts/kratos/README.md

```diff
@@ -64,7 +64,8 @@ A ORY Kratos Helm chart for Kubernetes
 | job.serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
 | job.shareProcessNamespace | bool | `false` | Set sharing process namespace |
 | job.spec.backoffLimit | int | `10` | Set job back off limit |
-| kratos.autoMigrate | bool | `false` | Enable the initialization job. Required to work with a DB |
+| kratos.automigration.enabled | bool | `false` | Enable the automigration. Required to work with a DB |
```

Comment:
This is autogenerated from comments in the values.yaml file, please do not modify manually :)